src/frontend/src/icp/utils/icrc.utils.ts

```diff
@@ -132,5 +132,5 @@ export const buildIcrcCustomTokenMetadataPseudoResponse = ({
 	];
 };
 
-export const isIcrcCustomToken = (token: Partial<IcrcCustomToken>): token is IcrcCustomToken =>
-	token.standard === 'icrc' && 'enabled' in token;
+export const isCustomTokenIC = (token: Partial<IcrcCustomToken>): token is IcrcCustomToken =>
```

Comment:
Please accept my apologies for commenting again. I am really sorry, but I just had a second thought. The term isCustmoTokenIC is actually misleading because there is no "ICP custom token" in the backend and in Oisy, there are ERC20 and ICRC custom tokens.

Can we call it something like "objContainsEnabled" or another more descriptive name? Any better ideas would also be appreciated.

Again, sorry for the inconvenience.

Reply:
eheh indeed... it was bothering me a little the name too...

what about `icTokenContainsEnabled`?

Reply:
Yeah way better thanks. Already approving so that you can go one without waiting on me.